package/requirements.txt

```diff
@@ -1,7 +1,9 @@
 packaging~=23.0
 kedro>=0.17.5
 ipython>=7.0.0, <9.0
-fastapi>=0.73.0, <0.96.0
+fastapi>=0.73.0,<0.200.0; python_version >= '3.8'
+fastapi>=0.73.0,<0.104.0; python_version < '3.8'
```

Comment:
Is Python < 3.8 still supported?

Reply:
yes, plan was to drop 3.7 once framework had but looks like you already have? We will plan in the upcoming sprint.